frontend/packages/component-library/src/cms/iframeEmbedContainer/stories/IframeEmbedContainer.story.tsx

```diff
@@ -0,0 +1,37 @@
+import type {Meta, StoryObj} from '@storybook/react';
+import {within, expect} from '@storybook/test';
+
+import IframeEmbedContainer, {IframeEmbedContainerProps} from '..';
+
+export default {
+  title: 'CMS/IframeEmbedContainer',
+  component: IframeEmbedContainer,
+  render: args => (
+    <div style={{height: '400px'}}>
+      <IframeEmbedContainer {...args} />
+    </div>
+  ),
+} as Meta<IframeEmbedContainerProps>;
+type Story = StoryObj<typeof IframeEmbedContainer>;
+
+const defaultArgs: IframeEmbedContainerProps = {
+  title: 'Iframe Title',
+  src: 'https://go.pardot.com/l/153401/2023-01-10/pqf2sr',
+};
+
+//
+// STORIES
+//
+export const Playground: Story = {
+  args: {
+    ...defaultArgs,
+  },
```

Comment:
Can you add this so it'll wait before capturing the Eyes baseline? Right now it's showing up as a blank screen (see screenshot below), and I think this should fix it. 
```suggestion
  },
  parameters: {
    eyes: {waitBeforeCapture: 4000},
  },
```
<img width="1616" alt="Screenshot 2025-03-06 at 8 20 54 AM" src="https://github.com/user-attachments/assets/69db9a0d-b0fb-4826-8d48-66393045a11d" />

Reply:
I'd actually prefer we use an internal data src rather than relying on Pardot (See above). In this way we can also remove the waitBeforeCapture